client/test/e2e/shared/components.js

```diff
@@ -32,3 +32,39 @@ exports.currencyInput = {
     return elm.getAttribute('ng-invalid');
   }
 };
+
+/**
+ * Object to help making end-to-end test,
+ * dateEditor.js.
+ * @public
+ */
+
+
+exports.dateEditor = {
```

Comment:
As discussed with @sfount , we will not use this component in patient registration so no need to update the e2e test for this unit.